library/alloc/src/vec/mod.rs

```diff
@@ -2760,6 +2795,15 @@ where
 #[cfg(not(test))]
 #[stable(feature = "vec_from_box", since = "1.18.0")]
 impl<T, A: Allocator> From<Box<[T], A>> for Vec<T, A> {
+    /// Convert a boxed slice into a vector.
+    /// No heap allocation is performed, and the items are not copied.
```

Comment:
I guess "are not copied" could be confusing to some. Can we just say that the vector takes over the box's allocation and elements? (Even avoiding the word move since the elements are not even touched even if formally moving ownership)

Reply:
We have similar wording in the docs but let's tweak so, if it makes more sense :)

Reply:
This better?

```

    /// Convert a boxed slice into a vector by transferring ownership of
    /// the existing heap allocation.
```